core/src/test/scala/org/locationtech/rasterframes/RasterFunctionsSpec.scala

```diff
@@ -972,4 +972,25 @@ class RasterFunctionsSpec extends TestEnvironment with RasterMatchers {
     val dResult = df.select($"ld").as[Tile].first()
     dResult should be (randNDPRT.localDefined())
   }
+
+  it("should check values isin"){
+    checkDocs("rf_local_is_in")
+
+    // tile is 3 by 3 with values, 1 to 9
+    val df = Seq((byteArrayTile, lit(1), lit(5), lit(10))).toDF("t", "one", "five", "ten")
```

Comment:
@vpipkt Change `Seq((byteArrayTile, lit(1), lit(5), lit(10)))` to `Seq((byteArrayTile, 1, 5, 10))` and the `java.lang.UnsupportedOperationException: No Encoder found for org.apache.spark.sql.Column` error should go away. You only want to use `lit`after the Dataframe is created.